api/service.py

```
@@ -822,269 +834,91 @@ async def get_anomaly_history(
-    return UplinkResponse(
-        status="sent",
-        ack_id=ack_id,
-        message=msg,
-        timestamp=datetime.now()
+# Authentication endpoints
+@app.post("/api/v1/auth/login", response_model=TokenResponse)
+async def login(request: LoginRequest):
+    """Authenticate user and return JWT token."""
+    auth_manager = get_auth_manager()
+    token = auth_manager.authenticate_user(request.username, request.password)
+    return TokenResponse(access_token=token, token_type="bearer")
```

Ruff (0.14.10)

Possible hardcoded password assigned to argument: "token_type"

(S106)